apps/src/templates/sectionProgress/standards/sectionStandardsProgressRedux.js

```diff
@@ -115,7 +115,24 @@ export function fetchStudentLevelScores(scriptId, sectionId) {
 }
 
 export function getNumberLessonsCompleted(state) {
-  return 5;
+  let lessonsCompleted = 0;
+  if (
+    state.sectionProgress.scriptDataByScript &&
+    state.scriptSelection.scriptId &&
+    state.sectionProgress.scriptDataByScript[state.scriptSelection.scriptId]
+  ) {
+    const stages =
+      state.sectionProgress.scriptDataByScript[state.scriptSelection.scriptId]
+        .stages;
+
+    stages.forEach(stage => {
+      const lessonCompletionStatus = getLessonCompletionStatus(state, stage.id);
+      if (lessonCompletionStatus.completed) {
+        lessonsCompleted += 1;
+      }
+    });
+  }
+  return lessonsCompleted;
```

Comment:
I wonder if we can re-use this here: https://github.com/code-dot-org/code-dot-org/blob/09ccb837f05fe71956ba0fd2c19865a9ca8d9de2/apps/src/templates/sectionProgress/standards/StandardsProgressTable.jsx#L156?

Reply:
They are doing different things. This new one is counting the number of lessons completed in the course. The one in the table is number of lessons completed for a specific standard.

Reply:
I'll rename the one in the table to make it more clear